persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/message/MessageEntity.kt

```diff
@@ -113,10 +115,11 @@ sealed class MessageEntity(
         VISIBLE, DELETED, HIDDEN
     }
 
+    @Serializable
     data class Mention(
-        val start: Int,
-        val length: Int,
-        val userId: QualifiedIDEntity
+        @SerialName("start") val start: Int,
```

Comment:
I'm not sure about putting this `Serializable` annotations in our entity models, I think they belong to our network dto's more, can we do it without ? 

Also, I didn't check that deep, but if we have it on other entities, I guess should be ok 😅

Reply:
We do have in other entities and it is done here explicitly just so it can decode from a JSON string into this data class 🤔